crypto/x509/x_crl.c

```diff
@@ -105,11 +105,15 @@ static int crl_set_issuers(X509_CRL *crl)
             gens = gtmp;
             if (!crl->issuers) {
                 crl->issuers = sk_GENERAL_NAMES_new_null();
-                if (!crl->issuers)
+                if (!crl->issuers) {
```

Comment:
When you're touching this could you please use `crl->issuers == NULL` instead?

Reply:
ok